app/src/test/java/org/mozilla/tv/firefox/fxa/FxaRepoTest.kt

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.tv.firefox.fxa
+
+import android.content.Context
+import io.mockk.MockKAnnotations
+import io.mockk.impl.annotations.MockK
+import io.mockk.verify
+import io.reactivex.observers.TestObserver
+import io.reactivex.subjects.BehaviorSubject
+import mozilla.components.concept.sync.OAuthAccount
+import mozilla.components.concept.sync.Profile
+import mozilla.components.service.fxa.manager.FxaAccountManager
+import org.junit.Before
+import org.junit.Test
+import org.junit.runner.RunWith
+import org.robolectric.RobolectricTestRunner
+
+@RunWith(RobolectricTestRunner::class)
```

Comment:
Since we aren't using real `Context`, we don't need to use Robolectric here.  I tried removing it, and it dropped down the execution time for the class from ~13.2 seconds to ~7.5s.